server/src/main/java/org/opensearch/monitor/jvm/JvmStats.java

```diff
@@ -299,7 +316,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         static final String PEAK_USED_IN_BYTES = "peak_used_in_bytes";
         static final String PEAK_MAX = "peak_max";
         static final String PEAK_MAX_IN_BYTES = "peak_max_in_bytes";
-
+        static final String USAGE_PERCENT = "usage_percent";
```

Comment:
Can this be confused with current usage vs usage after GC?

Reply:
Good question, I think it should not since the field is under `last_gc_stats` object, in this context, it should not be confusing. Also, the semantic of this field stays consistent between current usage and usage after GC.  What do you think?